documentation/ci-fix.md

```diff
@@ -236,6 +238,23 @@ TypeSpec using `OAuth2Auth` may generate the following OpenAPI:
 
 Which causes error `XmsEnumValidation`.  The recommended workaround is to add `omit-unreachable-types: true` to your `tspconfig.yaml`.
 
+### `ProvisioningStateMustBeReadOnly`
+
+The root cause is bugs in `azure-openapi-validator` and `oav`:
+
+https://github.com/Azure/azure-openapi-validator/issues/637
+https://github.com/Azure/oav/issues/848
+
+The recommended workaround is to add `use-read-only-status-schema: true` to your `tspconfig.yaml`.
+
+### `PatchBodyParameterSchema`
+
+The root cause is a bug in typespec-azure:
+
+https://github.com/Azure/typespec-azure/issues/383.
+
+The workaround in this issue should be applied to the spec TypeSpec sources.  The LintDiff error should **not** be ignored or suppressed.
```

Comment:
Should we re-word this to say that the workaround can be found in the linked issue? Just to make it a little more obvious for folks? I see Mark suggested a workaround in this comment: https://github.com/Azure/typespec-azure/issues/383#issuecomment-2038378469

Reply:
Fine with me, but can you suggest the exact alternate wording you would use?

Reply:
Maybe something like:
```suggestion
Please see the issue above for the suggested workaround and apply it directly to the spec TypeSpec sources.  The LintDiff error should **not** be ignored or suppressed.
```